apps/meteor/app/api/server/lib/rooms.ts

```diff
@@ -66,7 +83,7 @@ export async function findChannelAndPrivateAutocomplete({ uid, selector }) {
 
 	const userRoomsIds = Subscriptions.cachedFindByUserId(uid, { fields: { rid: 1 } })
 		.fetch()
-		.map((item) => item.rid);
+		.map((item: ISubscription) => item.rid);
```

Comment:
For consistency, this is not a complete ISubscription object, it's `Pick<ISubscription, 'rid'>`